app/_kong_plugins/jwt-signer/examples/validate-access-token-issuers.yaml

```
@@ -0,0 +1,15 @@
+description: Validates the not bofore claim in an access token.
```

```suggestion
description: |
  Validates the "not before" claim in an access token.
```
Reads oddly without any emphasis on the "not before".